File: Treinamento/Python/Exercícios/Estrutura_de_decisão/Questão06.py
print('*** Maior número ***')

n1 = int(input('Digite o primeiro número: '))
n2 = int(input('Digite o segundo número: '))
n3 = int(input('Digite o terceiro número: '))

if n1 > n3 and n1 > n2:
    print('Maior número: ' + str(n1))
elif n2 > n3 and n2 > n1:
    print('Maior número: ' + str(n2))
elif n3 > n2 and n3 > n1:
    print('Maior número: '+ str(n3))

File: Treinamento/Java/Estrutura_de_decisão/Questão06.java
import java.util.Scanner;
public class Questão06 {
    public static void main(String[] args) {
        Scanner s;
        s = new Scanner(System.in);

        System.out.print("Digite o primeiro número: ");
        int n1 = s.nextInt();

        System.out.print("Digite o segundo número: ");
        int n2 = s.nextInt();

        System.out.print("Digite o terceiro número: ");
        int n3 = s.nextInt();

        if (n1 > n2 && n1 > n3) {
            System.out.println("Maior número: " + n1);
            if (n3 > n2) {
                System.out.println("Menor número: " + n2);
            } else {
                System.out.println("Menor número: " + n3);
            }
        } else if (n2 > n1 && n2 > n3) {
            System.out.println("Maior número: " + n2);
            if (n1 > n3) {
                System.out.println("Menor número: " + n3);
            } else {
                System.out.println("Menor número: " + n1);
            }
        } else {
            System.out.println("Maior número: " + n3);
            if (n2 > n1) {
                System.out.println("Menor número: " + n1);
            } else {
                System.out.println("Menor número: " + n2);
            }
        }
    }
}
            

